ru-ru/rust-ru.html.markdown

```diff
@@ -0,0 +1,311 @@
+---
+language: rust
+contributors:
```

Comment:
Please add a line here -
```
lang: ru-ru
```

This is to ensure the file is parsed as a Russian translation of an existing page. Without this line, your translation would make a duplicate entry on the website homepage.

Reply:
Line must be added after `language: rust`